app_grim_reefer/Kconfig

```diff
@@ -0,0 +1,161 @@
+# Copyright (c) 2021 Nordic Semiconductor ASA
+# SPDX-License-Identifier: Apache-2.0
+#
+# This file is the application Kconfig entry point. All application Kconfig
+# options can be defined here or included via other application Kconfig files.
+# You can browse these options using the west targets menuconfig (terminal) or
+# guiconfig (GUI).
+
+menu "Zephyr"
+source "Kconfig.zephyr"
+endmenu
+
+menuconfig LSM6DSL
+	bool "LSM6DSL I2C/SPI accelerometer and gyroscope Chip"
+	default y
+	depends on DT_HAS_ST_LSM6DSL_ENABLED
+	select I2C if $(dt_compat_on_bus,$(DT_COMPAT_ST_LSM6DSL),i2c)
+	select SPI if $(dt_compat_on_bus,$(DT_COMPAT_ST_LSM6DSL),spi)
+	help
+	  Enable driver for LSM6DSL accelerometer and gyroscope
+	  sensor.
+
+if LSM6DSL
+
+choice LSM6DSL_TRIGGER_MODE
+	prompt "Trigger mode"
+	help
+	  Specify the type of triggering to be used by the driver.
+
+config LSM6DSL_TRIGGER_NONE
+	bool "No trigger"
+
+config LSM6DSL_TRIGGER_GLOBAL_THREAD
+	bool "Use global thread"
+	depends on GPIO
+	select LSM6DSL_TRIGGER
+
+config LSM6DSL_TRIGGER_OWN_THREAD
+	bool "Use own thread"
+	depends on GPIO
+	select LSM6DSL_TRIGGER
+
+endchoice
+
+config LSM6DSL_TRIGGER
+	bool
+
```

Comment:
Wipe the config lines here.

Reply:
fixed in [314d58d](https://github.com/RIT-Launch-Initiative/FSW/pull/52/commits/314d58d64e4811e7a7bd31817aa276b685ae3c5b)